src/workerd/io/compatibility-date.capnp

```diff
@@ -584,7 +584,7 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   nodeJsZlib @59 :Bool
       $compatEnableFlag("nodejs_zlib")
       $compatDisableFlag("no_nodejs_zlib")
-      $experimental;
```

Comment:
There is a compatibility-date-test.c++ that can/should be updated to verify this. Look at line 182 in the current test. That verifies that the `nodejs_compat_v2` flag is appropriately set on the compat date, the `nodejs_zlib` flag can be checked there also.